Babylon/utils/macro.py

```diff
@@ -48,9 +48,9 @@ def step(self,
         :param run_if: run this step only if this argument is True.
         :return: The original macro used for method chaining
         """
-        if self._status != self.STATUS_OK or not run_if:
-            logger.warning(f"Skipping command {' '.join(command_line)}...")
-            return self
+        # if self._status != self.STATUS_OK or not run_if:
```

Comment:
Even if it is a work around i'd rather prefer a new boolean option: continue_on_error or something like that you could use during your test without corrupting the real usage.